src/components/Rounds/Leaderboard.jsx

```diff
@@ -24,7 +25,7 @@ export default class extends React.Component {
   getRank = (points) => {
     if (!this.state.leaderboard.length) return 0;
 
-    let scores = Array.from(new Set(this.state.leaderboard.map(team => team.points)));
+    let scores = Array.from(new Set(this.state.leaderboard.map(team => team.points))).sort((a,b)=>b-a);
```

Comment:
`this.state.leaderboard` is already sorted, it doesn't need to be sorted here again.

Reply:
Wrote this because in download the teams were listed in team order so scores set were not in depending order, having a safety measure.

Reply:
Well, it's always sorted everything the `leaderboard` state is set.
https://github.com/manipalutsav/frontend/blob/635cc9d9886e648df1a766f50414529b89b4cfac/src/components/Rounds/Leaderboard.jsx#L36-L38
https://github.com/manipalutsav/frontend/blob/635cc9d9886e648df1a766f50414529b89b4cfac/src/components/Rounds/Leaderboard.jsx#L44-L46
Anyway, doesn't matter. Approving.